lib/kubeclient/common.rb

```diff
@@ -308,11 +308,20 @@ def get_entity(klass, resource_name, name, namespace = nil, options = {})
       new_entity(result, klass)
     end
 
-    def delete_entity(resource_name, name, namespace = nil)
+    def delete_entity(resource_name, name, namespace = nil, delete_options: {})
```

Comment:
There is nothing specific here for delete options right? we should rename in that case